# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/cgit/scripts-path.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- cgit-1.2.3/Makefile.vanilla	2024-01-30 12:50:13.024491931 +0100
+++ cgit-1.2.3/Makefile	2024-01-30 12:52:04.804000158 +0100
@@ -2,7 +2,7 @@
 
 CGIT_VERSION = v1.2.3
 CGIT_SCRIPT_NAME = cgit.cgi
-CGIT_SCRIPT_PATH = /var/www/htdocs/cgit
+CGIT_SCRIPT_PATH = /usr/share/webapps/cgit
 CGIT_DATA_PATH = $(CGIT_SCRIPT_PATH)
 CGIT_CONFIG = /etc/cgitrc
 CACHE_ROOT = /var/cache/cgit
